src/Processing.js

```diff
@@ -1158,6 +1158,16 @@
       return new XML(p, uri);
     };
 
+    /**
+     * Processing 2.0 function for creating XML elements from string
+     *
+     * @param {String} xml the XML source code
+     *
+     * @return {XML} An XML object representation of the input XML markup.
+     */
+    p.parseXML = function(xmlstring) {
+      return new XML.parse(xmlstring);
```

Comment:
looking at the code, this should not actually return an XML element, so now I'm really curious why the unit test passes.